index.js

```diff
@@ -65,24 +98,62 @@ class ABA {
      * @param options.description {String} Description of this file entries. Up to 12 chars.
      * @param [options.date=Date.now()] {Date|String|Number} Date to be processed.
      * @param [options.time] {Date|String|Number} Time to be processed. Should be ignored according to the specs.
+     * @param [options.schemas] {Object} Custom schemas
+     * @param [options.customHeaderData] {Object} Custom data for custom header, warning, will overwrite standard field if name will be the same
```

Comment:
I find this `custom...` option too difficult to understand. Let's rethink. This should be more user friendly.